edge-modules/TwinTester/docker/linux/amd64/Dockerfile

```diff
@@ -1,17 +1,26 @@
 ARG base_tag=2.1.13-alpine3.10
+FROM azureiotedge/azureiotedge-runtime-base:1.1-linux-amd64 as builder
```

Comment:
why rename dockerfile for messages analyzer to twin tester?

Reply:
This diff shows weirdly due to renamed files. What I have done here is created new dockerfiles that sets up rocksdb. 

Also. for arm32 and arm64 there are base dockerfiles required that we will use in place of the ones under SimulatedTemperatureSensor. These new base images only exist under TestAnalyzer, as I did not want to duplicate.